components/sections/About.tsx

```diff
@@ -120,20 +123,41 @@ export default function About(): React.ReactElement {
           <Flex justifyContent="center" textAlign="center" marginBottom="32px">
             <HeadlinePrimary>Ways to experience CUSEC</HeadlinePrimary>
           </Flex>
-          <Flex
-            align={["center", "center", "center", "stretch"]}
-            direction={["column", "column", "column", "row"]}
-            wrap="wrap"
-            justify="center"
+          <Grid
+            width="100%"
+            gridTemplateColumns={[
+              "repeat(1, 1fr)",
+              "repeat(1, 1fr)",
+              "repeat(1, 1fr)",
+              "repeat(1, 1fr)",
+              "repeat(2, 1fr)",
+            ]}
+            gridTemplateAreas={[
+              gridColumn,
+              gridColumn,
+              gridColumn,
+              gridColumn,
+              gridBox,
+            ]}
```

Comment:
I finally realized after all this time we could specify 5 styles, instead of just 4, for the breakpoints :woman_facepalming: 

i.e. the `breakpoints = ["30em", "48em", "62em", "80em"];` specified in `theme.js` means:

1. 0-30em
2. 30-48em
3. 48-62em
4. 62-80em
5. 80+ em

Reply:
No way LOL. I'm glad you posted this! 😂